### PR TITLE
Feature/gsap

### PR DESCRIPTION
**Pull request by joanafbrito**
# Description drawing

- Install GSAP
- Add Css Style to How to page
- Add gsap animation for the `PokemonDetails`

Next steps :

Still have code to be clean
Add more animations

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [x] console.log()
- [x] dev tools
- [ ] cypress testing

# Checklist:

- [ ] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/HowTo/HowTo.css [hunk 1]
```diff
@@ -1,5 +1,17 @@
 .help-container {
```
Comment (hayleyw7) [inline src/components/HowTo/HowTo.css:1]: Looks great! I love the red-blue.

File: src/components/PokemonDetails/PokemonDetails.js [hunk 2]
```diff
@@ -18,11 +23,31 @@ const PokemonDetails = ({ foundPokemon, getPokemonImage, clearPokemon }) => {
...
+      {
+        opacity: 1,
+        x: 70,
+        y: 10,
+        ease: Power3.easeOut,
+        repeat: true
+      }
+    )
+
+    console.log('here')
```
Comment (Matt-Roden) [inline src/components/PokemonDetails/PokemonDetails.js:44]: Great job! Nice work finding a way to actually use the gsap animations :)

File: src/components/PokemonDetails/PokemonDetails.js [hunk 3]
```diff
@@ -18,11 +23,31 @@ const PokemonDetails = ({ foundPokemon, getPokemonImage, clearPokemon }) => {
       const res = await fetch(url);
       const pokemonDetails = await res.json();
       setPokemonDetails(pokemonDetails);
+      
+      gsap.fromTo('.single-pokemon-pic' , .8,
```
Comment (hayleyw7) [inline src/components/PokemonDetails/PokemonDetails.js:27]: This is so cool! I can't wait to add more!

**Review by hayleyw7 — APPROVED** [review]

File: src/components/Search/Search.css [hunk 4]
```diff
@@ -13,7 +13,7 @@
   font-size: 25px;
   text-align: center;
   border-radius: 15px;
-  border: 2px solid #3b4cca;
+  border: 2px solid #3b4ccaad;
```
Comment (Matt-Roden) [inline src/components/Search/Search.css:16]: This is a nice edit, looks great

**Review by Matt-Roden — APPROVED** [review]
Looks great, thank you!

**Pull request merged**